src/azure-cli-core/tests/test_rp_auto_reg.py

```diff
@@ -0,0 +1,63 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import unittest
+import json
+import time
+import mock
+
+from azure.cli.testsdk import (
+    JMESPathCheck,
+    ResourceGroupPreparer,
+    ScenarioTest)
+
+from azure.cli.core.commands import _check_rp_not_registered_err
+
+
+# pylint: disable=line-too-long
```

Comment:
the only line requires this is line 34, can you put it there? Additionally, that line itself can be broken into multiple lines.